chain/network/src/routing.rs

```diff
@@ -824,8 +826,8 @@ impl RoutingTable {
         trace!(target: "network", "Update routing table.");
 
         self.peer_forwarding = self.raw_graph.calculate_distance();
-
-        let now = Clock::instant();
+        let clock = &mut Clock {};
+        let now = clock.instant();
```

Comment:
These `&mut Clock {}` feel odd, perhaps we should keep `::instant` to be an associated fn (without self)?

Reply:
Done